tests/scripts/test_plot.py

```diff
@@ -0,0 +1,43 @@
+
+
+import blocks.scripts.plot as plot
+
+from collections import OrderedDict
+from unittest import skipIf
+#from blocks.log import TrainingLog
+#from tests import silence_printing
+
+try:
+    from pandas import DataFrame
+    PANDAS_AVAILABLE = True
+except:
+    PANDAS_AVAILABLE = False
+
+
+def some_experiments():
+    """Create some 2 dummy experiments."""
+    experiments = OrderedDict()
+    experiments['exp0'] = DataFrame()
+    experiments['exp0']['col0'] = (0,1,2)
+    experiments['exp0']['col1'] = (3,4,5)
+    experiments['exp1'] = DataFrame()
+    experiments['exp1']['col0'] = (6,7,8)
+    experiments['exp1']['col1'] = (9,9,9)
+    return experiments
+
+
+#@silence_printing
+@skipIf(not PANDAS_AVAILABLE, "pandas not installed")
```

Comment:
Good point, I should probably turn it into one. Just habit I guess, it wasn't a decorator in Pylearn2. I'll change it in a bit.